tests/test_alphabet.py

```diff
@@ -6,7 +6,7 @@
 def _test_esm1b(alphabet):
     import torch
 
-    batch_converter = alphabet.get_batch_converter()
+    batch_converter = alphabet.get_batch_converter(truncate=True)
```

Comment:
You can leave these tests as-is (without argument), but add a test specifically specifying `truncation_seq_length=10` for example, and making sure the output makes sense (proper truncation, padding and EOS behave as expected when there is trunction or no truncation)

Reply:
I added a test to check the behavior of truncated sequences